circle.yml

```diff
@@ -2,6 +2,7 @@ dependencies:
   cache_directories:
     - ~/.pyenv/versions/3.4.3
     - ~/coala-atom/node_modules
+    - ~/nltk_data
```

Comment:
Not sure if this needs to be in a separate commit, ci was failing on this PR due to it so I think it should be alright here. Still failing, I might need to add `python3 -m nltk.downloader punkt maxent_treebank_pos_tagger averaged_perceptron_tagger` in the script.